test/index.test.js

```diff
@@ -29,7 +29,7 @@ describe('mongodb-connection-model', function() {
       it('should return the correct URL for the driver', function() {
         var c = new Connection();
         assert.equal(c.driver_url,
-          'mongodb://localhost:27017/?slaveOk=true&ssl=prefer');
+          'mongodb://localhost:27017/?slaveOk=true');
```

Comment:
@rueckstiess Background is PR #109 https://mongodb.github.io/node-mongodb-native/driver-articles/mongoclient.html#connection-configuration but happy to wait for @imlucas to weigh in on the direction to proceed here.